db/schema.rb

```diff
@@ -146,7 +146,7 @@
     t.decimal "growout_trough"
     t.decimal "length"
     t.decimal "mass"
-    t.string "gonad_score"
+    t.decimal "gonad_score"
```

Comment:
⚠️ This is a schema change in the PR, currently without a corresponding database migration to change the column type in existing databases.

Reply:
Oops, good catch @robbkidd!

@wesleyeewong Can you fix this?  I think this PR is ready to merge aside from that.

Also I know it's been a really long-running PR with a ton of requests for change, and you've put a lot of really patient work into it, so if you want any help getting the last little bit done, please ask :heart: